docs/contributing/contributing-to-decred.md

```diff
@@ -0,0 +1,43 @@
+
+As with many other open source software development projects, people working on Decred do so autonomously and voluntarily. People contribute to Decred because they want the project to succeed and there is some aspect they are personally attached to, enough to want to spend some of their time working on it. The Treasury has funds to pay people for this work, once they demonstrate that they are capable of adding significant value in a particular domain.
+
+The strength of the open source approach to production is that individuals efficiently allocate themselves to the tasks they are most interested in and apt for. Contributors are typically not keen to spend time managing or directing others, so this model tends to only suit people who enjoy working independently in a self-directed manner.
+
+
+
+---
+
+## <img class="dcr-icon" src="/img/dcr-icons/Introductions.svg" /> Becoming a paid Contractor
+
+Decred has a Treasury fund to pay contributors who work effectively to advance the project. To become a paid contractor one must demonstrate that one's work is of value to the project. This [blog post](https://blog.decred.org/2017/07/25/Decred-Recruiting/) by Project Organizer Jake Yocom-Piatt is still the definitive guide to becoming a contractor.
+
+The people who are best placed to judge whether a new contributor would make a good addition to a team are the people who are already working in that team. New contributors who produce good work will be invited to become a paid contractor.
+
+At present, this process is informal, but a formal process for granting a Decred Contractor Clearance (DCC) that allows an individual to bill for their work has been [proposed](https://proposals.decred.org/proposals/fa38a3593d9a3f6cb2478a24c25114f5097c572f6dadf24c78bb521ed10992a4) and approved by stakeholders. This process will involve three established contractors in the relevant domain agreeing that a new contributor should be invited to become a contractor. This DCC model will go live once the [contractor management system](https://github.com/decred/contractor-mgmt) is operational.
+
+As part of becoming a paid contractor, you will agree an hourly rate at which you can bill for your work on the project. Contractors are also eligible for a portrait to be shown on the [contributors page](https://decred.org/contributors/) of decred.org.
+
+The [Contributor Compensation](contributor-compensation.md) page has details of how billing and payments work.
+
+Questions about becoming a contributor or contractor can be asked in the #Recruiting channel - available on bridged [chat platforms](https://decred.org/community). The first steps are generally to read the [blog post](https://blog.decred.org/2017/07/25/Decred-Recruiting/), explore the sub-project(s) you're most interested in contributing to (typically on [GitHub](https://github.com/decred/)), and talk to the people who work on those already.
```

Comment:
1) `#recruiting` (lowercase); 2) it is 'room' in Matrix slang (I have no hard opinion on which to use, if 'channel' is more common go with it)